tests/unittests/TensorsTest.cpp

```diff
@@ -994,10 +994,41 @@ TEST(Tensor, randomizeFused_Float) {
 }
 
 /// Test randomizing a Fused tensor with Float16 scale/offsets.
-TEST(Tensor, randomizeFused_Foat16) {
+TEST(Tensor, randomizeFused_Float16) {
   testRandomizeFused(ElemKind::UInt8FusedFP16QTy);
 }
 
+/// Check that getting and setting fused tensors works correctly.
+template <typename ScaleOffsetT>
+static void testGetSetFusedScaleOffset(ElemKind fusedKind) {
+  Tensor T(fusedKind, {10, 10}, 1.0, 0);
+  auto TH = T.getHandle<uint8_t>();
+  for (size_t i = 0; i < 10; i++) {
+    TH.setFusedScaleOffsetInRow<ScaleOffsetT>(i, i, i);
+  }
+  for (size_t i = 0; i < 10; i++) {
+    ScaleOffsetT scale, offset;
+    std::tie(scale, offset) = TH.getFusedScaleOffsetFromRow<ScaleOffsetT>(i);
+    EXPECT_EQ(scale, (ScaleOffsetT)i);
+    EXPECT_EQ(offset, (ScaleOffsetT)i);
+  }
+}
+
+/// Test getting and setting fused offsets from UInt8FusedQTy.
```

Comment:
"fused offsets" ->"fused offsets and scales".  Same for other 2 tests' comments.